backend/Makefile

```diff
@@ -75,7 +75,7 @@ db/dump_schema:
 ifeq ($(DEPLOYMENT_STAGE),test)
 	docker-compose exec database pg_dump --schema-only --dbname=corpora --username corpora
 else
-	$(eval DB_PW = $(shell aws secretsmanager get-secret-value --secret-id corpora/backend/${DEPLOYMENT_STAGE}/database --region us-west-2 | jq -r '.SecretString | split(":") | .[-1] | split("@") | .[0]'))
+	$(eval DB_PW = $(shell aws secretsmanager get-secret-value --secret-id corpora/backend/${DEPLOYMENT_STAGE}/database --region us-west-2 | jq -r '.SecretString | match(":([^:]*)@").captures[0].string'))
```

Comment:
DRY it?

Reply:
sigh, can, was planning on some of that for more intensive rethinking of these Makefiles, but that's for later, this was just to make it actually work 😄

Reply:
'Rethinking' meaning, moving these utilities _out_ of Makefiles, I hope? 🙏 🤠